py/DREAM/Output/SPIShardPositions.py

```diff
@@ -59,6 +125,105 @@ def calcRadialCoordinate(self, shards=None, t=None):
         thetap = np.arctan2(yp[t,shards],xp[t,shards])
 
         return rhop, thetap
+
+
+    def plotAtTime(self, t=-1, shards=None, ax=None, show=None):
+        """
+        Plot the pellet shards over the poloidal cross-section at a given time.
+        """
+        black = (87/255, 117/255, 144/255)
+        red = (249/255, 65/255, 68/255)
+
+        genax = ax is None
+
+        if genax:
```

Comment:
Same comment about `genax` as above, i.e. is it necessary to define a separate variable?